ext/curl/interface.c

```diff
@@ -909,6 +909,18 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SSL);
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SSPI);
 
+#if LIBCURL_VERSION_NUM >= 0x071306 /* Available since 7.19.6 */
```

Comment:
These definitions should be part of the existing version checks below (other CURL_VERSION constants are in there as well)